server/api/utils/email.ts

```diff
@@ -0,0 +1,63 @@
+import nodemailer from "nodemailer";
+
+interface Item {
+    name: string;
+    quantity: number;
+}
+async function sendApprovalEmail(firstName: string, lastName: string, email: string, item: Item) {
+    const transporter = nodemailer.createTransport({
+        service: "gmail",
+        auth: {
+            user: "blueprintpregnancycentre@gmail.com",
```

Comment:
Will this be changed? If so, add TODO or make a reminder somewhere

Reply:
Updated.